server/src/main/java/org/apache/iotdb/db/qp/physical/crud/GroupByPlan.java

```diff
@@ -33,6 +33,8 @@
   // if it is left close and right open interval
   private boolean leftCRightO = true;
 
+  private boolean byTime = false;
```

Comment:
The GroupByPlan is actually GroupByTimePlan, so no need to add a byTime field, we can rename the class.